docs/reference/action_list.rst

```diff
@@ -136,71 +141,75 @@ Available types and associated options
-|           | body           | Add body parameter to email link.                                     |
-+-----------+----------------+-----------------------------------------------------------------------+
-| percent   |                | Renders value as a percentage.                                        |
-+-----------+----------------+-----------------------------------------------------------------------+
-| string    |                | Renders a string.                                                     |
-+-----------+----------------+-----------------------------------------------------------------------+
-| text      |                | See 'string'                                                          |
-+-----------+----------------+-----------------------------------------------------------------------+
-| html      |                | Renders string as html                                                |
-+-----------+----------------+-----------------------------------------------------------------------+
-| time      |                | Renders a datetime's time with format ``H:i:s``.                      |
-+-----------+----------------+-----------------------------------------------------------------------+
-| trans     | catalogue      | Translates the value with catalogue ``catalogue`` if defined.         |
-+-----------+----------------+-----------------------------------------------------------------------+
-| url       | url            | Adds a link with url ``url`` to the displayed value                   |
-+           +----------------+-----------------------------------------------------------------------+
-|           | route          | Give a route to generate the url                                      |
-+           +                +                                                                       +
-|           |   name         | Route name                                                            |
-+           +                +                                                                       +
-|           |   parameters   | Route parameters                                                      |
-+           +----------------+-----------------------------------------------------------------------+
-|           | hide_protocol  | Hide http:// or https:// (default: false)                             |
-+-----------+----------------+-----------------------------------------------------------------------+
++-----------+---------------------+-----------------------------------------------------------------------+
+| Type      | Options             | Description                                                           |
++===========+=====================+=======================================================================+
+| actions   | actions             | List of available actions                                             |
```

Comment:
Looking again, I think this way is more consistent with the other rows (since "show", "edit", "history" and "delete" are the first subnode for the "actions" option, while "link_parameters" is a child of them):
```
+---------+----------------------------------+---------------------------+
| actions | show, edit, history, delete, etc | Any of available actions. |
+         +                                  +                           +
|         |   link_parameters                | Route parameters          |
+---------+----------------------------------+---------------------------+
```

Reply:
With you example, I understand that edit is an option for the type `actions` 
![image](https://user-images.githubusercontent.com/9052536/77941494-8a6b6900-72ba-11ea-8dfa-0a4cb16afe6b.png)

But this is not ! The type `actions` has an option called `actions`:
![image](https://user-images.githubusercontent.com/9052536/77941418-690a7d00-72ba-11ea-80a7-b5638321e6a2.png)

The first column `actions` is not the options, it's the type of the field ; and the second one is the name of the options. So I have to show that there is an options called `options`.

Reply:
Oh, I see, then I think it would be something like:
```
+---------+------------------------------------+---------------------------------------------------------------------------------------- +
| actions | actions                            | Can be any of the available actions (``show``, ``edit``, ``history``, ``delete``, etc). |
+         +                                    +                                                                                         +
|         |   edit                             | The name of the action.                                                                 |
+         +                                    +                                                                                         +
|         |     link_parameters                | Route parameters.                                                                       |
+---------+------------------------------------+-----------------------------------------------------------------------------------------+
```
For the other types, "options" rows are only splitted when they have different options for the type, but as you stated, the only option here is "actions" ("edit" & co. are children).